Restore from discovery holds (#2600)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md

Public folders that have been deleted by users or administrators are stored in the public folder dumpster located in `\NON_IPM_SUBTREE\DUMPSTER_ROOT`. Deleted folders are preserved here until the time retention period is over.

> For scenarios where public folder contents are put on hold using retention policy, the folders reomved from `\NON_IPM_SUBTREE\DUMPSTER_ROOT` are preserved under `\NON_IPM_SUBTREE\DiscoveryHolds` till the time retention hold is over.

Reviewer: ```suggestion
For scenarios where public folder contents are put on hold using retention policy, the folders removed from `\NON_IPM_SUBTREE\DUMPSTER_ROOT` are preserved under `\NON_IPM_SUBTREE\DiscoveryHolds` until the retention hold period is over.
```

> Any folders preserved in the public folder dumpster or under DiscoverHolds folder can be restored using EXO PowerShell. Restoring the public folder will restore all subfolders and items present in the folder.

Reviewer: ```suggestion
Any folders preserved in the public folder dumpster or under the DiscoverHolds folder can be restored using EXO PowerShell. Restoring the public folder will restore all subfolders and items present in the folder.
```

    Set-PublicFolder -Identity \NON_IPM_SUBTREE\DUMPSTER_ROOT\DUMPSTER_EXTEND\RESERVED_1\RESERVED_1\9f32c468-4bc2-42aa-b979-16a057394b2f\PF1 -Path \
    ```

> In this another example, to restore a public folder named `Sales` to the root of the public folder tree:

Reviewer: ```suggestion
    The following alternate example restores a public folder named `Sales` to the root of the public folder tree:
```

>     ```PowerShell

Reviewer: ```suggestion
    ```PowerShell
```
